Make the link input field large

chmst opened the pull request:
When I want to know which is the link behind a menu item, I have to inspect the code because the field shows only the first and uninteresting part of the link. This small change makes ist possible to see the whole link.
![menu-item](https://cloud.githubusercontent.com/assets/1035262/8055852/e6964500-0ea2-11e5-934b-d2a78e7d79c0.PNG)

chmst committed:
Make the link input field large

When I want to know which is the link behind a menu item, I have to inspect the code because teh field shows only the first and uninteresting part of the link. This small change makes ist possible to see the whole link.

Fedik commented:
:+1: 
**test** good

anibalsanchez commented:
@test ok<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7141">issues.joomla.org/joomla-cms/7141</a>.</sub>

zero-24 commented:
RTC based on testing. Thanks :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7141">issues.joomla.org/joomla-cms/7141</a>.</sub>

brianteeman commented:
This needs to be fixed in hathor as well

bembelimen commented:
I really like the idea of a bigger link field, but what about a little event like: the input stays normal and while a user hovers with the mouse, the input field extends...

chmst committed:
Fixed: for hathor

@brianteenman: thank you for remembering

@bembelimen: attractive idea ..for future versions

chmst committed:
Update item.xml

Fedik commented:
@bembelimen I do not want to do any additional action just to see what in the field :smile:

zero-24 commented:
@test successful also for hathor see:

![with_the_patch_hathor](https://cloud.githubusercontent.com/assets/2596554/8067787/397cfd16-0ef0-11e5-9ac7-d0cb4be03b50.PNG)

I agree with @Fedik to keep this simple as possible so I move it back to RTC :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7141">issues.joomla.org/joomla-cms/7141</a>.</sub>

merka commented:
OK.
